sim/verilator/run_test.sh

```diff
@@ -3,6 +3,7 @@
 set -e
 set -x
 
+export CONDA_BUILD_SYSROOT=/Users/runner/sdk/MacOSX10.9.sdk
```

Comment:
is this something we should have on a generic level?

Reply:
This SDK is downloaded and unpacked in this custom location by download_sdk.sh script. Without pointing to it test won't pass: https://github.com/litex-hub/litex-conda-eda/runs/1662891928?check_suite_focus=true
Wrong SDK is used

Reply:
Piotr means generic level as e.g. workflow level. We can't have this env variable defined for whole workflow, because Conda overrides these variables and creates separate env for running tests.